src/databricks/labs/ucx/account/metastores.py

```diff
@@ -48,6 +48,8 @@ def assign_metastore(
         # set the default catalog using the default_namespace setting API
         if default_catalog is not None:
             self._set_default_catalog(workspace_id, default_catalog)
+        logger.info(f"Metastore {metastore_id} assigned to workspace {workspace_id}")
+        logger.info("Run the 'create-ucx-catalog' command to create the UCX catalog in case it does not exist")
```

Comment:
Do we want to explain why?

```suggestion
        logger.info("Run the `create-ucx-catalog` command to create the UCX catalog for tracking your migration progress")
```

"in case it does not exists" is not required IMO